components/tools/OmeroPy/src/omero/plugins/admin.py

```diff
@@ -365,9 +375,20 @@ def __init__(this, name, help, wait=False):
                 help="Print out which files would be deleted")
             x.add_argument(
                 "data_dir", type=DirectoryType(),
-                help="omero.data.dir directory value (e.g. /OMERO")
+                help="omero.data.dir directory value e.g. /OMERO")
             x.add_login_arguments()
 
+        removepyramids.add_argument(
```

Comment:
perhaps add the `wait` arguments? https://github.com/openmicroscopy/openmicroscopy/pull/5608/files#diff-57f0d9cb121726c674f02fd0bc23bd1cR220

Reply:
I do not have access to that commit in the workflows branch
I can open  a follow up PR when this is merged into develop